exampleSite/config.yaml

```diff
@@ -147,6 +147,9 @@ Languages:
     # --- navigation menu ---
     menu:
       main:
+        - name: Home
+          weight: 10
+          url: .
```

Comment:
Would the _Home_ URL ever be a page that was not the landing page of the website? I feel like it is better to keep the config simple and always include the homepage from the layout. Do you agree?